f5_openstack_agent/lbaasv2/drivers/bigip/network_service.py

```diff
@@ -500,8 +534,11 @@ def get_bigip_net_short_name(self, bigip, tenant_id, network_name):
     @staticmethod
     def get_neutron_net_short_name(network):
         # Return <network_type>-<seg_id> for neutron network
-        net_type = network['provider:network_type']
-        net_seg_key = network['provider:segmentation_id']
+        net_type = network.get('provider:network_type', "")
```

Comment:
Would we rather return a possibly malformed net short name or raise an exception if the get doesn't find this key?